src/experimental-layers/package.json

```diff
@@ -81,9 +82,5 @@
     "uglify-js": "^2.6.1",
     "webpack": "^2.4.0",
     "webpack-dev-server": "^2.4.0"
-  },
-  "peerDependencies": {
-    "react": "0.14.x - 16.x",
-    "react-dom": "0.14.x - 16.x"
```

Comment:
Why are these gone?

Reply:
> Why are these gone?

The layers themselves have no dependencies on React. They are just ES6 classes. 

Although deck.gl itself still requires React, we have an experimental API in deck.gl v5 that allows deck.gl to be used without React.